man/systemd.network.xml

```diff
@@ -1015,6 +1015,18 @@
             </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>Type=</varname></term>
+          <listitem>
+            <para>The Type identifier for special route types, which can be
+            <literal>blackhole</literal> packets are discarded silently,
+            <literal>unreachable</literal> packets are discarded and the ICMP message host unreachable is generated,
+            <literal>prohibit</literal> packets are discarded and the ICMP message communication administratively
+            prohibited is generated. Defaults to unset.
```

Comment:
Why isn't there "unicast" to choose from?

Hmm, there should always be a way that if somebody sets Type=prohibit at the top of the [Route] section he should be able to revert to the original value again with a later line, and currently that's not possible, is it? Unless Type=unicast is available default behaviour cannot be restored with a line further down...

Reply:
I think you mean we should add the `unicast` ?

Reply:
yes